### DISCUSSION TITLE
How to change color depth

### DISCUSSION THREAD

**Discussion by ghost** [tannerhelland/PhotoDemon#593] [Q&A]
How can I change the color depth of an image?
E.g. from 24 bit to 8 bit?

**Comment by tannerhelland** [comment]
Hello @WriteEraseMan .  There are two ways to convert an image to 8-bit color in PhotoDemon:

1. `Adjustments > Map > Palette map`
2. `Effects > Stylize > Palette`

Both of those menus launch the same tool.  The tool provides many options for *how* the image is converted to 8-bit color.

Some formats (e.g. PNG) also allow you to directly convert to 8-bit in the save options screen, if that's easier for you.  This is typically referred to as `indexed color mode`.

Just a quick note - PhotoDemon automatically manages color-depth for you.  It will always export images as 8-bit if it can (for example, if the image has 256 colors or less and the target file format supports it).
